examples/demo/site/themes/bootstrap/page.tpl

```diff
@@ -40,7 +40,7 @@
 	<!-- Page search -->
 	<div class="row">
 		<div class="col-md-2 col-md-offset-9">
-			<form action="{$site_url/}gcse" class="search-form" id="gcse_search_form">
+			<form action="{$site_url/}gcse20" class="search-form" id="gcse_search_form">
```

Comment:
Ok here, we may need to find alternative .. since we are mixing cms search and specific module.
I guess, we should have a "search" region, and the theme will place it as expected.
And for custom theme, we could hardcode the gcse20 url ...